packages/lib/server-only/user/get-all-users.ts

```diff
@@ -0,0 +1,37 @@
+import { prisma } from '@documenso/prisma';
+
+type getAllUsersProps = {
+  page: number;
+  perPage: number;
+};
+
+export const findUsers = async ({ page = 1, perPage = 10 }: getAllUsersProps) => {
+  const [users, count] = await Promise.all([
+    await prisma.user.findMany({
+      select: {
+        id: true,
+        name: true,
+        email: true,
+        roles: true,
+        Subscription: {
+          select: {
+            id: true,
+            status: true,
+            planId: true,
+            priceId: true,
+            createdAt: true,
+            periodEnd: true,
+          },
```

Comment:
Looks like an `include: { Subscription: true } }` is more suitable here 😸 

Great idea with limiting the size of the query though by omitting columns we don't care about.

Reply:
If I do that, wouldn't it pull all the fields? In this case, I omit `customerId`, `userId`, and `updatedAt`.

Reply:
Yeah while beneficial those fields aren't _large_ enough to justify worrying about it. Happy to keep it for now but a rule of thumb is that unless it's adding a bunch of data to be sent across the wire we shouldn't worry about including all columns.

Simplicity => Perf

Reply:
Ok, implemented the feedback. Thanks!